bin/tedi

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# This wrapper runs the Tedi build system inside Docker.
+#
+# https://github.com/elastic/tedi/blob/master/README.md
+
+set -euo pipefail
+
+tedi_version="${TEDI_VERSION:-0.13}"
+tedi_debug="${TEDI_DEBUG:-false}"
+elastic_version=$($(dirname ${BASH_SOURCE})/elastic-version)
```

Comment:
It seems to be the only line that makes the script depend on `pyhton3`.
I'm using MacOS, I'm not a Python expert, and configuring my dev env to use pyhon2.7 or python3 is sometimes frustrating.

```
bash-3.2$ ./bin/tedi build --fact=image_tag:7.0.0-SNAPSHOT
env: python3: No such file or directory
```
it could be interesting to be able to test a build command with just docker as a requirement like this:
```
./bin/tedi build --fact=image_tag:7.0.0-SNAPSHOT --asset-set=remote_snapshot
```

Reply:
Good point. I think that `elastic-version` is probably a lot more complicated than it needs to be now. For example, it parses JSON because we used to have multiple versions per branch. We don't do that any more, so the input file, `version.json` is just a single-element object. Maybe it could be a string and we could re-write `elastic-version` as a few lines of shell.

Your thoughts, @mgreau, @Conky5?